front/lib/api/assistant/generation.ts

```diff
@@ -39,6 +43,7 @@ export type ModelMessageType = {
 
 export type ModelConversationType = {
   messages: ModelMessageType[];
+  tokensUsed: number;
```

Comment:
Let's add it next to it not inside of it as this type is really meant to be what we send the model?

Reply:
fixed